sagemathinc/cocalc
# support for .Rtex files, generalizing what was done for .Rnw

see #2997

this generalizes .Rnw extra logic to also work for .Rtex files.

testing:webapp: 

1. create a .Rtex file (smc_pyutil contains a suitable example)
2. check that the pdf preview eventually shows up (this here patchs a utilitiy function, which derives the pdf filename)
3. forward/inverse search should work
4. R code syntax errors should show up in the error box
